encoder/vaapiencoder_base.cpp

```diff
@@ -322,7 +326,7 @@ SurfacePtr VaapiEncoderBase::createSurface(const SharedPtr<VideoFrame>& frame)
 
 void VaapiEncoderBase::fill(VAEncMiscParameterHRD* hrd) const
 {
-    hrd->buffer_size = m_videoParamCommon.rcParams.bitRate * m_videoParamCommon.rcParams.windowSize/1000;
+    hrd->buffer_size = m_videoParamCommon.rcParams.bitRate * 4;
```

Comment:
4 seems a magic number. Can we use a macro for this?
Why original code use windowSize, new code do not use it?